ex02.ex

```diff
@@ -53,8 +53,11 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
-
+  first2equal = fn (list) ->
```

Comment:
## -2

Here you have a single function body with three lines of code.

But think in terms of functions. What would a function look like that takes a list where the first two values are equal and returns true:

~~~ elixir
  first2equal = fn
                  [ a, a | _ ]              -> true
                  other when is_list(other) -> false
                end
~~~
This is now a single function with two heads. The first is a function that recognizes the list with two equal terms, and the second recognizes all others